### DISCUSSION TITLE
Manually edit similar named TV/Movies - E.g The Witcher

### DISCUSSION THREAD

**Discussion by figlizzard12** [sct/overseerr#364] [General]
Is there a way to manually update exisiting shows/movies to point to the correct show? The Witch series is showing as the 2002 version instead of the latest 2019 one. Will only be affected when new seasons release, but would be good to have a way to force the files to point to which version you have.

**Comment by samwiseg0** [comment]
That is determined by the TMDB ID can your verify that you have the correct TMDB ID in Plex for that show?
